docs/setup/permissions.md

```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
 
 #### External Storage Modification
 
-Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need 
+### Read your contacts
+
+In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
+need access to your contacts.
+
+### Modify your contacts
+
+In order to note that a contact has been emailed, we need write-access.
+
+## Storage
+
+### Read the contents of your USB storage
+### Modify or delete the contents of your USB storage
+
+Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need
 access to do so.
```

Comment:
We no longer request the storage permission (either for reading or writing). For all file system access outside of our app-private directory we use the storage access framework that lets users pick files/locations.